src/ProjectNeon/Assets/Scripts/BattleTurnWrapUpPhase.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BattleTurnWrapUpPhase : MonoBehaviour
+{
+    [SerializeField] private GameEvent battleTurnWrapUpStarted;
+    [SerializeField] private GameEvent battleTurnWrapUpFinished;
+    [SerializeField] private GameEvent playerTurnConfirmed;
+    [SerializeField] private GameEvent turnStarted;
+
+    private void OnEnable()
+    {
+        playerTurnConfirmed.Subscribe(StartBattleTurnPhase, this);
```

Comment:
Every Event subscription must have a corresponding `OnDisable` `Unsubscribe`, other the GameEvent is corrupted.